src/components/timeline.js

```diff
@@ -0,0 +1,385 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useEffect, useRef, useState } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { withStyles, makeStyles } from '@material-ui/styles';
+import Slider from '@material-ui/core/Slider';
+import ClockIcon from '../images/icons/clock.svg';
+import TextField from '@material-ui/core/TextField';
+
+import { FormattedMessage } from 'react-intl';
+import {
+    connectNotificationsWebsocket,
+    fetchMergesByProcessAndDate,
+} from '../utils/api';
+import {
+    currentMergesList,
+    updateIgmStatus,
+    updateMergeDate,
+} from '../redux/actions';
+import { IgmStatus } from './merge-map';
+
+// eslint-disable-next-line
+const marks = [
```

Comment:
Why do you need this? Does it mean we can only display these hours?

Reply:
Just to define the time slots from 00:00 to 23:59, on the other hand if you test with a file for example with a time 22:52 it will be displayed in the right place, so we can display any time.